crates/wysiwyg/src/dom/nodes/container_node.rs

```diff
@@ -140,6 +140,20 @@ where
         ret
     }
 
+    pub fn remove_child_and_own_hierarchy(
```

Comment:
Can we find a better name e.g. `remove_child_and_take_grandchildren` ? (Or something better??)